app/views/candidate_interface/personal_details/immigration_status/_shared_form.html.erb

```diff
@@ -22,7 +22,7 @@
   <%= f.govuk_text_area(
     :right_to_work_or_study_details,
     label: { text: t('page_titles.immigration_status'), size: 'l' },
-    hint: { text: 'For example, “I have permanent residence”' },
+    hint: { text: 'Enter your visa type or immigration status - for example, “Student visa, expires 28 August 2025”, “Indefinite leave to remain” or “Family visa”' },
```

Comment:
incorrect use of a hyphen

Reply:
cc @duncanjbrown

Reply:
🚓

Reply:
It appears gov uk recommends this sort of usage:
![image](https://github.com/DFE-Digital/apply-for-teacher-training/assets/35870975/4be6379c-e43b-4c06-931f-519edae7594f)

https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style

I'll amend.

Reply:
It's not a hyphen, it's a dash to introduce a parenthesis. See example in 'Bullet points and steps' in the style guide: "do not use more than one sentence per bullet - use commas or dashes to expand on an item". I try to avoid using full stops in hint text so I used a dash. It's fine if you want to update as per your example, though. So: Enter your visa type or immigration status. For example, “Student visa, expires 28 August 2025”, “Indefinite leave to remain” or “Family visa”.